### ISSUE TITLE
Talkback announces wrong state on radio button selection

### ISSUE THREAD

**Issue by artesgo** [angular/components#17319]
#### Reproduction

Use StackBlitz to reproduce your issue: https://stackblitz.com/fork/components-issue

Steps to reproduce:
1. Load https://material.angular.io/components/radio/examples on android chrome browser
2. Start Talkback
3. Select 'Autumn' option from the first example (talkback reads 'Selected')
4. Navigate to 'Summer' (talkback reads 'Summer radio button not selected')
5. Select 'Summer' (talkback reads 'Not selected')
6. Select 'Autumn' again (talkback reads 'Selected')
 
 
#### Expected Behavior

What behavior were you expecting to see?
When selecting a **previous** radio button, talkback should announce the correct state, 'Selected'

#### Actual Behavior

What behavior did you actually see?
When selected a **previous** radio button, talkback announces it's previous state

#### Environment

- Angular: latest
- CDK/Material: latest
- Browser(s): Mobile Chrome, Android WebView
- Operating System (e.g. Windows, macOS, Ubuntu): Android

Can close this for now, Issue appears to be from Android's talkback.
[link to issuetracker](https://issuetracker.google.com/issues/119763080)

**Comment by mmalerba** [comment]
I just tested this on and it seems to work as expected on:
Pixel 3
Android 10
Chrome 83
Angular Material 10.0.1
